XRTK.Oculus/Packages/com.xrtk.oculus/Editor/ValidateOculusControllerMappings.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEditor;
+using UnityEngine;
+using XRTK.Oculus.Controllers;
+using XRTK.Utilities.Editor;
+
+namespace XRTK.Oculus
+{
+    [InitializeOnLoad]
+    public class ValidateOculusControllerMappings
+    {
+        private static Type[] controllerTypes = new[] { typeof(OculusTouchController) };
+        [SerializeField]
```

Comment:
cannot serialize static fields

Reply:
That's leftover stuff I forgot to clear out. My bad